docs/tutorials/basics/part1.md

```diff
@@ -100,42 +100,104 @@ of **changes** to the view, but cannot inspect its current contents.
 
 Click the PLAY button to run the pipeline.
 
-## Step 2. Populate tables manually
+## Step 2. Insert data
 
-:::info UNDER CONSTRUCTION
+When the pipeline is running it can process incoming changes. Let's try just that via ad-hoc queries.
 
-This section is under construction.
+Open Ad-hoc query tab and insert in the input field the following statement:
 
-:::
+```sql
+INSERT INTO VENDOR (id, name, address) VALUES
+(1, 'Gravitech Dynamics', '222 Graviton Lane'),
+(2, 'HyperDrive Innovations', '456 Warp Way'),
+(3, 'DarkMatter Devices', '333 Singularity Street');
+```
+
+Press `Enter` or click the Play button to submit the query. You will see the result containing the number of inserted rows: 3.
+
+![Query to insert vendor data](basics-part1-1.png)
+
+Now let's check the state of the table with a quick select query that we insert into the empty input field below the result from the first query:
+
+```sql
+SELECT * FROM VENDOR
+```
+
+![Inserted vendor data](basics-part1-2.png)
+
+Yep, everything is in order. Let's fill the other table:
+
+```sql
+INSERT INTO PART (id, name) VALUES
+(1, 'Flux Capacitor'),
+(2, 'Warp Core'),
+(3, 'Kyber Crystal')
+```
+
+```sql
+INSERT INTO PRICE (part, vendor, price) VALUES
+(1, 2, 10000),
+(2, 1, 15000),
+(3, 3, 9000);
+```
+
+The ad-hoc queries we run are executed by the same pipeline that handles the stream processing, but instead of using our incremental computation engine they are evaluated against the latest snapshot of the pipeline's tables and views.
+
+Enough with the static SQL, let's see Feldera's incremental computation in action!
```

Comment:
From here on, I think we could be more pedagogical. 

* Don't send the user switching between change stream and sql tabs. Let's first explain incremental view maintenance by using SQL only (you have most of this text below, so this is mostly changing the structure).
  * Explain that whenever you insert into a table, Feldera incrementally updates all vies
  * Direct the user to `SELECT *` from one of the views.
  * Modify one of the inputs and `SELECT *` again to see the modified view

* Introduce the change stream:
  * explain that in Feldera you can observe a stream of changes to the views, not just the current snapshot
  * Switch to the change stream tab, tick the boxes.
  * Switch back to SQL, modify one of the tables again
  * Observe deltas in the Change stream
(BTW, having to switch between the tabs is probably bad ergonomics; we should think about a UI that combines the two views on the same screen)

* Inserting JSON data directly in the change stream. I suggest we don't go there in this tutorial.  The focus here is on understanding the basics, and this feature isn't really needed for that.